tests/plugins/content_checker_test.py

```diff
@@ -0,0 +1,134 @@
+import json
+import pytest
+from detect_secrets.core.potential_secret import PotentialSecret
+from detect_secrets.plugins.content_checker import ContentChecker
+from testing.mocks import mock_file_object
+
+secretExpected = True
+noSecretExpected = False
+java_source_file_tests = [
+		{
+			"name":    "Java assignment spanning newline",
+			"content": 'String jwtSecret = "dasdasdasdasdasdasdasdasdasddaasdasdasdasdsdhashdsahdhsadhhasdhahdhashdhahdhah";',
+			"want":    secretExpected,
+		},
+		# {
```

Comment:
It looks like some of the values in `java_source_file_tests` are commented out. Was this intentional?